Fixed early sell bug and Added auto-sell functionality

Harikeshk7 opened the pull request:
Due to way that conditional statements were structured, bug occurred where some stock would sell 0 shares at the beginning. Fixed this bug as well as added functionality to sell all owned stocks when the function is done running

Harikeshk7 committed:
Fixed early sell bug and Added auto-sell functionality
Due to way that conditional statements were structured, bug occurred
where some stock would sell 0 shares at the beginning. Fixed this bug
as well as added functionality to sell all owned stocks when the
function is done running

ttaka808 reviewed (changes requested):
Remove the unused code and comments

Harikeshk7 committed:
Included 'codebase cleanup change'
Previous bug fix and functionality update removed earlier codebase
cleanup changes. Have included the earlier changes as well

Harikeshk7 commented:
> Remove the unused code and comments

Done
